packages/hint/src/lib/@types/browserslist.d.ts

```diff
@@ -1,4 +0,0 @@
-declare module 'browserslist' {
-    function browserslist(match?: string | string[]): string[];
```

Comment:
I removed this since browserslist now provides its own `.d.ts` file. It was masking the more complete types from browserslist, including the existence of `loadConfig` which I needed to use to have browserslist look for known config files before falling back to our new defaults.